homeassistant/components/bond/fan.py

```diff
@@ -199,10 +199,8 @@ async def async_turn_on(
 
     async def async_set_preset_mode(self, preset_mode: str) -> None:
         """Set the preset mode of the fan."""
-        if preset_mode != PRESET_MODE_BREEZE or not self._device.has_action(
-            Action.BREEZE_ON
-        ):
-            raise ValueError(f"Invalid preset mode: {preset_mode}")
+        if not self._device.has_action(Action.BREEZE_ON):
+            raise ServiceValidationError(f"Invalid preset mode: {preset_mode}")
```

Comment:
@gjohansson-ST I believe the code on line 203 is unreachable. The capability attribute is set on creation only and also used to determine the fan has this feature. So I believe it is safe to remove the check.